en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+  
+AllJoyn Explorer and its documentation (how to install, to use…) can be find [here](http://ms-iot.github.io/content/en-US/win10/AllJoyn.htm).
+
+![ZigBeeHardware]({{site.baseurl}}/images/ZigBee/ZigBeeHardware.png)
+
+## Set up steps
+1. Install the required tools and driver listed in the prerequisites on your Windows 10 desktop
+(see their respective documentations to figure out how to proceed).
+2. Configure the XBee module
+3. Let your device join your ZigBee network
+4. set up your Raspberry Pi2 (if you target that device)
+5. Deploy ZigBee adapter
+
+>Note that in Windows 10, when a machine has __multiple AllJoyn modern applications__ that __need to interact__ on the same machine, the user must __add a loopback exemption__ for these modern applications. Consequently, if you run both the ZigBee adapter and AllJoyn Explorer on the same machine you will need to add a loopback exemption for these 2 applications. This isn’t needed for application you run from Visual Studio 2015. Note that when deploying an application from Visual Studio 2015, the loopback exemption is for the lifetime of the installed application. Meaning that you can launch the app directly (not from Visual Studio 2015) afterwards and it will have the loopback exemption.
+
+Set up loopback exception: 
+ 1. Find the installation folder of the modern application for which you want to enable the loopback exemption. It is located at "C:\Users\\*username*\AppData\Local\Packages"
+ 
+ ![LoopBackException]({{site.baseurl}}/images/ZigBee/LoopBackException.png)
+ 2. Copy the installation folder name which is also the application ID.
+ 3. Run the following command from an elevated command prompt:
+`CheckNetIsolation LoopbackExempt -a -n=installation-folder-name`
+ 4. Restart your applications.
+
+## Configure your XBee ZigBee module using XCTU tool
+Please look at the tool help to get more details about the tool (https://docs.digi.com/display/XCTU/XCTU+Overview).
+
+![XBeeConfig1]({{site.baseurl}}/images/ZigBee/XBeeConfig1.png)
+
+![XBeeConfig2]({{site.baseurl}}/images/ZigBee/XBeeConfig2.png)
+
+## Let ZigBee devices join your ZigBee network
+Once the XBee ZigBee module has been configured you can build your ZigBee network and let your ZigBee devices join. In order to do that you just need to power up your ZigBee devices. ZigBee Light Link (aka ZLL) and Home Automation devices will, by default and if not already part of a ZigBee network, try to join a ZigBee network which permit join is enabled. Since the XBee ZigBee module has been configured to always enable permit join, the ZigBee devices will join your network. 
+
+You can verify that devices have by using “network discovery” feature of XCTU tool.
+
+![ZigBeeJoinNetVerif]({{site.baseurl}}/images/ZigBee/ZigBeeJoinNetVerif.png)
+
+## Set up your Raspberry Pi2
+1. perform initial set up as instructed [here](http://ms-iot.github.io/content/en-US/win10/SetupRPI.htm)
```

Comment:
Please capitalize the First letter of the first word of each of these steps to match with other documentation.